core-web/libs/dotcms-scss/jsp/scss/dotcms.scss

```diff
@@ -53,14 +53,36 @@
 
 html {
     font-family: $font-default;
-    font-size: $font-size-default;
     color: $font-color-base;
 }
 
 body {
     margin: 0;
     padding: 0;
 }
+span,
+label,
+div,
+dl,
+dt,
+dd,
+ul,
+ol,
+li,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+button,
+textarea,
+p,
+blockquote,
+th,
+td {
+    font-size: $font-size-md;
+}
```

Comment:
Should we modify the default size of these tags? @dotCMS/frontend

Reply:
We have too, because the root font size remains 16px